src/main.tsx

```diff
@@ -1,12 +1,249 @@
-import './index.css';
-
-import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
+import { createGlobalStyle } from 'styled-components';
+
+import Page1 from './Page1.tsx';
+import Page2 from './Page2.tsx';
+import Page3 from './Page3.tsx';
+import Page4 from './Page4.tsx';
+import Page5 from './Page5.tsx';
+import Page6 from './Page6.tsx';
+
+const ResetStyle = createGlobalStyle`
+  * {
+    box-sizing: border-box;
+  }
+  
+  html,body,div,span,applet,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,
+  small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,
+  aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,
+  nav,output,ruby,section,summary,time,mark,audio,video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+  }
+  article,aside,details,figcaption,figure,footer,header,hgroup,menu,
+  nav,section {
+    display: block;
+  }
+  body {
+    line-height: 1;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  }
+  ol,
+  ul {
+    list-style: none;
+  }
+  blockquote,
+  q {
+    quotes: none;
+  }
+  blockquote:before,
+  blockquote:after,
+  q:before,
+  q:after {
+    content: "";
+    content: none;
+  }
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+  }
+  
+  button {
+    background: inherit; 
+    border:none; 
+    border-radius:0;
+    padding:0; 
+    cursor:pointer;
+  }
+  
+  a {
+    color: inherit;
+    text-decoration: none;
+    outline: none;
+  }
+  
+  a:visited {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  input {
+    -webkit-appearance: none; /* Safari and Chrome */
+       -moz-appearance: none; /* Firefox */
+            appearance: none;
+  }
+  
+  input[type='number']::-webkit-inner-spin-button,
+  input[type='number']::-webkit-outer-spin-button {
+      -webkit-appearance: none;
+         -moz-appearance: none;
+              appearance: none;
+  }
+  
+  input:focus {
+    outline: none;
+  }`;
```

Comment:
styled-reset을 사용하면 조금 더 깔끔하게 정리할 수 있을 것 같습니다. 보통 styled component와 같이 많이 사용합니다.

```suggestion
import { createGlobalStyle } from 'styled-components';
import reset from 'styled-reset';

const ResetStyle = createGlobalStyle`
createGlobalStyle`
  ${reset}

  html {
    font-size: 62.5%;
  }

  body, button, input, select, textarea {
	  font-family: "Pretendard Variable", Pretendard, Roboto, "Apple SD Gothic Neo", "Noto Sans KR", sans-serif;
  }

  * {
    box-sizing: border-box;
  }
`;

```